agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/telemetry/TelemetryChannel.java

```diff
@@ -73,19 +74,17 @@ public class TelemetryChannel {
 
   private static final AppInsightsByteBufferPool byteBufferPool = new AppInsightsByteBufferPool();
 
-  private static final int MAX_STATSBEAT_ERROR_COUNT = 10;
-
   private final OperationLogger operationLogger;
   private final OperationLogger retryOperationLogger;
 
   // TODO (kryalama) do we still need this AtomicBoolean, or can we use throttling built in to the
   //  operationLogger?
   private final AtomicBoolean friendlyExceptionThrown = new AtomicBoolean();
 
-  private final AtomicInteger statsbeatErrorCount = new AtomicInteger();
-  private final AtomicBoolean statsbeatShutdown = new AtomicBoolean();
+  private final AtomicInteger statsbeatUnableToReachBreezeCounter = new AtomicInteger();
```

Comment:
can we rename it to statsbeatUnableToReachServerCounter? breeze is for internal.

Reply:
I think "Server" is vague, e.g. which server did it reach, a proxy server? I don't mind renaming it to "IngestionService" though then the var name becomes even longer. This is internal code, and the behavior is specific to Breeze, so I think Breeze is ok, but will rename to "IngestionService" if you really prefer that.

Reply:
breeze is going away.. do you plan to update this to a newer system name? I'm ok with IngestionService.

Reply:
do we know that the new system will be backwards compatible with breeze? if not, then we will have to revisit this anyways at that time

Reply:
i think so. let's use ingestionService and not reveal our internal system name in a public repo?

Reply:
> not reveal our internal system name in a public repo

we use the term breeze in several other places already if that is your concern

Reply:
oops.. we did. then yeah it's ok to leave as it is..